pkg/loop/internal/ocr2/config.go

```diff
@@ -7,45 +7,47 @@ import (
 	libocr "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 	"google.golang.org/grpc"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/chainreader"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/core"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/net"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb"
 	"github.com/smartcontractkit/chainlink-common/pkg/types"
 )
 
 var (
-	_ types.ConfigProvider = (*configProviderClient)(nil)
-	_ GRPCClientConn       = (*configProviderClient)(nil)
+	_ types.ConfigProvider = (*ConfigProviderClient)(nil)
+	_ core.GRPCClientConn  = (*ConfigProviderClient)(nil)
 )
 
-type configProviderClient struct {
-	*ServiceClient
+type ConfigProviderClient struct {
+	*core.ServiceClient
```

Comment:
Unexport?

Reply:
Oh wait are all of these embedded to inherit methods? :thinking: Maybe no problem then, but still feels a bit odd.